src/commons/XMLParser/XMLParserHelper.ts

```diff
@@ -167,7 +167,11 @@ const makeQuestions = (task: XmlParseStrTask): [Question[], number] => {
 
 const makeMCQ = (problem: XmlParseStrCProblem, question: BaseQuestion): IMCQQuestion => {
   const choicesVal: MCQChoice[] = [];
-  const solution = problem.SNIPPET ? problem.SNIPPET[0].SOLUTION : undefined;
+  const snippet = problem.SNIPPET;
+  // FIXME: I think `XmlParseStrCProblem` type definition is incorrect
+  // FIXME: Remove `as unknown as keyof typeof snippet` when fixed
+  // @ts-expect-error broken type definition to be fixed above
+  const solution = snippet ? snippet[0 as unknown as keyof typeof snippet].SOLUTION : undefined;
```

Comment:
This file is used solely in MissionControl, which is marked as low priority/ deprecation. Consider starting an issue tracker for MissionControl?

Reply:
I think we can do that, though if we are removing it soon, the issue tracker might not be needed. Still waiting on Prof @martin-henz to reply on whether any of the profs still wants it to be kept.